frontend/packages/react-app/src/components/organisms/WalletDistributionTargets/index.tsx

```diff
@@ -56,7 +56,7 @@ const WalletDistributionTargets: React.FC<TargetsProps> = ({
                 <Typography variant={"body1"}>Format: </Typography>
                 <Box ml={2}>
                   <Typography variant={"body1"} color={"secondary"}>
-                    {`{"targets": ["walletaddress1", "walletaddress2"], "type": "address"}`}
+                    {`{"targets": ["walletaddress1", "walletaddress2"]}`}
```

Comment:
A logic that checks type field is contained in API, so it should be still displayed. 
https://github.com/iroiro/iroiro/blob/1aaa3166a37443a1f423be63835ae296e57aef59/functions/merkle-tree/functions/input-generator/src/app.ts#L35-L37